libs/utils/trace.py

```diff
@@ -874,6 +867,27 @@ def getClusterActiveSignal(self, cluster):
 
         return cluster_active
 
+    def addEventsDeltas(self, df, col_name='len'):
```

Comment:
It's now a more generic API... let's match the default with the method name, i.e. `s/len/delta`

Reply:
I think the main reason I did that is because the overutilized dataframe used to be built with that 'len' column, and I didn't want to break anything. But that's not really an issue, I can just specify 'len' for overutilized instead of making it the norm for everyone else.